gossip/components/processor.c

```diff
@@ -40,7 +40,8 @@ static retcode_t process_transaction_bytes(processor_t const *const processor,
   retcode_t ret = RC_OK;
   bool exists = false;
   struct _iota_transaction transaction = {.metadata.snapshot_index = 0,
-                                          .metadata.solid = 0};
+                                          .metadata.solid = 0,
+                                          .loaded_columns_mask = 0};
```

Comment:
Random thought: if we were to add some more fields to be defaulted, or if you think 3 are already forgettable, maybe we should add a `DECLARE_TX` macro or something

Reply:
Alright